src/Console_TEST.cc

```diff
@@ -470,13 +489,14 @@ TEST_F(Console_TEST, ColorErr)
 /// \brief Test Console::Verbosity
 TEST_F(Console_TEST, Verbosity)
 {
-  EXPECT_EQ(common::Console::Verbosity(), 1);
```

Comment:
It looks like this was testing the default verbosity? Is this now different?

Reply:
This line has been restored in [f3d168c](https://github.com/gazebosim/gz-common/pull/615/commits/f3d168cfdc7eb4a8575edc4dfbf24cd8a98a5c52).